node-cli/index.js

```diff
@@ -33,6 +44,9 @@ async function run() {
     "Please log in to your Realm account or register as a new user."
   );
 
+
```

Comment:
Question: are the extra linefeeds intentional? It might be easier to read through without them.